### ISSUE TITLE
Review annotations to GO:0034649 histone demethylase activity (H3-monomethyl-K4 specific GO:0034648 histone demethylase activity (H3-dimethyl-K4 specific) 

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-annotation#3871]
The proposal has been made to obsolete
GO:0034649 histone demethylase activity (H3-monomethyl-K4 specific)
and
GO:0034648 histone demethylase activity (H3-dimethyl-K4 specific)

The reason for obsoletion is to align with RHEA and EC: these reactions are sub-reactions of both RHEA and EC.
These terms have 5 and 13 EXP annotations, respectively.
GO:0034648 is mapped to RHEA:60216, but this is part of RHEA:60208, which corresponds to GO:0034647.

Suggestions to move existing annotations are here: https://docs.google.com/spreadsheets/d/1mlEpZji59zgeTQaxan0Ky6hE8B7rEjrrGRg9RpfcatE/edit#gid=0

Group | Number of annotations to fix 
-- | --
CACAO | 2
CAFA | 1
MGI | 5
PomBase | 1
SGD | 1
WB | 4

**Comment by srengel** [comment]
SGD done.

**Comment by vanaukenk** [comment]
WB done.

**Comment by hdrabkin** [comment]
mgi done

**Comment by ValWood** [comment]
pombase done.

**Comment by Antonialock** [comment]
uniprot done

**Comment by Antonialock** [comment]
@pgaudet there are still two annotations left to fix?

**Comment by pgaudet** [comment]
Yes - by CACAO - @Antonialock do you have permissions on these?

**Comment by Antonialock** [comment]
no, if I should then I can ask Alex to give me permission? @pgaudet

**Comment by Antonialock** [comment]
@pgaudet Alex says the annotation is from an external source and that  This has to be fixed by CACAO people

**Comment by pgaudet** [comment]
@dsiegele Can you please review these 2 CACAO annotations? 
https://docs.google.com/spreadsheets/d/1mlEpZji59zgeTQaxan0Ky6hE8B7rEjrrGRg9RpfcatE/edit#gid=0
